GitHubGraphQl.ps1

```diff
@@ -225,18 +228,18 @@ function Invoke-GHGraphQl
 
                 $requestId = ''
 
-                if ($ex.Response.PSTypeNames[0] -eq 'System.Net.Http.HttpResponseMessage')
-                {
-                    $requestId = ($ex.Response.Headers | Where-Object -Property Key -eq 'X-GitHub-Request-Id').Value
-                }
-                elseif ($ex.Response.PSTypeNames[0] -eq 'System.Net.HttpWebResponse')
+                if ($ex.Response.PSTypeNames[0] -eq 'System.Net.HttpWebResponse')
```

Comment:
I'm curious about this re-ordering of the exception checking.  Were you encountering a bug that you were trying to work around? If so, does that same bug manifest in `Invoke-GHRestMethod` as well?

Reply:
Sorry Howard, I'm not sure what you mean. Can you clarify?

Reply:
In 2fd31e0720ab25f44509ffba03d369711dca2d58, you changed from first checking `System.Net.Http.HttpResponseMessage` to first checking `System.Net.HttpWebResponse`, as part of a commit that refined your exception handling.  I was curious what bug you had encountered that required the re-ordering.  It was curiosity, not critique.

I was then separately wondering if the issue/reasoning for caused you to re-order how you checked those exceptions would be relevant for `Invoke-GHRestMethod` as well...but upon re-reading _that_ code, I realized that this part of the code is unique now to `Invoke-GHGraphQl`

Reply:
I think it was just so that the PowerShell 5 exception was checked first, and then the PowerShell 6 one, to match up the order with the comment above, nothing more than that.